07 Form/src/components/member/pageContainer.tsx

```diff
@@ -3,15 +3,20 @@ import * as toastr from 'toastr';
 import { memberAPI } from '../../api/member';
 import { MemberEntity } from '../../model';
 import { MemberPage } from './page';
+import PropTypes from 'prop-types';
 
 interface State {
   member: MemberEntity;
 }
 
-export class MemberPageContainer extends React.Component<{}, State> {
-  constructor() {
-    super();
+interface Props {
+  history: PropTypes.object.isRequired;
```

Comment:
This is OK, it is using the default React typing from prop-types library. I wonder though if there would be a better way to do this. @brauliodiez, any suggestions?

Reply:
For type `history` you have to:

```javascript
import { History } from 'history';
...
interface Props {
  history: History;
}
```

Reply:
Since is typescript we should not use prop-types